nemo/collections/nlp/parts/nlp_overrides.py

```diff
@@ -102,8 +106,7 @@ def configure_ddp(self):
                 device_ids=device_ids,
                 output_device=device_ids[0],
                 process_group=app_state.data_parallel_group,
-                find_unused_parameters=False,
```

Comment:
I think we just need to remove this line? I don't think we want to set our own default.

Reply:
OK, that's fine too. It's the easier option but the default will then change in NeMo if it ever changes in PTL. I'll assume that's OK and will update right now.

Reply:
Done